
From: Guillaume Thouvenin <guillaume.thouvenin@bull.net>

Function wdtpci_init_one() in file wdt_pci.c generates a warning when
compiling the watchdog driver.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/char/watchdog/wdt_pci.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/char/watchdog/wdt_pci.c~watchdog-fix-warning-defined-but-not-used drivers/char/watchdog/wdt_pci.c
--- 25/drivers/char/watchdog/wdt_pci.c~watchdog-fix-warning-defined-but-not-used	2004-08-09 22:01:54.746427712 -0700
+++ 25-akpm/drivers/char/watchdog/wdt_pci.c	2004-08-09 22:01:54.750427104 -0700
@@ -674,8 +674,8 @@ out:
 out_misc:
 #ifdef CONFIG_WDT_501_PCI
 	misc_deregister(&temp_miscdev);
-#endif /* CONFIG_WDT_501_PCI */
 out_rbt:
+#endif /* CONFIG_WDT_501_PCI */
 	unregister_reboot_notifier(&wdtpci_notifier);
 out_irq:
 	free_irq(irq, &wdtpci_miscdev);
_
